java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Appetizer/AppetizerRequestBody.java

```diff
@@ -7,7 +7,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Data
-public class FoodTruckRequestBody2 {
+public class AppetizerRequestBody {
```

Comment:
Do we need to repeat the word appetizer in all these fields?

Reply:
Not Sure what you mean. But i changed the name(AppetizerRequestBody) because the original name(FoodTruckRequestBody2) was wrong.

Reply:
the field names, not the class/object name

Reply:
<img width="290" alt="image" src="https://user-images.githubusercontent.com/78369712/189446412-97a20374-4cb6-4df3-8563-a6b69679625a.png">

Reply:
Ops misunderstood Let me fix that